pages/docs/user-docs/build-docker-module.md

```diff
@@ -0,0 +1,43 @@
+---
+title: docker bootstrap module
+sidebar: user_docs
+permalink: build-docker-module
+folder: docs
+toc: false
+---
+
+This module allows you to build a container from an existing container hosted on Docker Hub or another Docker registry. 
+
+{% include toc.html %}
+
+## Overview
+You can use an existing container image as your "base," and then add customization. For example, instead of using the `debootstrap` module to start an Ubuntu container, you might use the official Ubuntu container from Docker Hub as a base.  You could do the same with CentOS, Debian, Arch, Suse, Alpine, BusyBox, etc.  Or maybe you want to build a container that uses CUDA and cuDNN to leverage the GPU, but ou don't want to install from scratch.  You can start with one of the `nvidia/cuda` containers and install your software on top of that.
+
+## Keywords
+```
+Bootstrap: docker
+```
+The **Bootstrap** keyword is always mandatory. It describes the bootstrap module to use.
+```
+From: /username/container:tag
+```
+The **From** keyword is mandatory.  It specifies the container to use as a base. In the case of some official registries (ubuntu for example) the `username` is optional.  The `tag` is also optional and will default to `latest`
+```
```

Comment:
This isn't totally correct, the terms we've been using are:

`<registry>/<namespace>/<container>:<tag>@<digest>`

For the registry, we default to index.docker.io. For namespace we default to library, and for tag (you got it!) latest. This would be good to link here to the page that describes the different formulations of the URI, AND the new page I added that describe how to customize the bootstrap with environment, etc. If you don't think the second makes sense here, the bottom would work too!

Reply:
Thanks for setting me straight here.  I also think it's a good idea to link to pages with more info, but I'm not sure which pages you mean.  Could you give me a few pointers please?

Reply:
Here is the docker uri (on the docker page): http://singularity.lbl.gov/docs-docker#how-do-i-specify-my-docker-image

and the "additional stuffs" page: https://github.com/singularityware/singularityware.github.io/blob/docs/2.4/pages/docs/user-docs/build-environment.md

Reply:
Thank you!